package.json

```diff
@@ -10,13 +10,14 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.1",
     "react-scripts": "4.0.3",
     "web-vitals": "^1.0.1"
```

Comment:
## OPTIONAL COMMENT:
- [ ] It would be better if we add a negative test case to our project, this helps improve the performance of the app